subsystem-drive/dto/drive-dto.hpp

```diff
@@ -1,33 +1,46 @@
 #pragma once
+
 namespace sjsu::drive
 {
     // note: postfix commands and arguments can be seperated into different files
     struct drive_commands
     {
-        int angle = 0; // -180 <--> 180
+        char mode = 'D';
         int speed = 0; // -100 <--> 100
+        int angle = 0; // -180 <--> 180
         int wheel_orientation = 0;
-        char mode = 'D';
+
+        void Print()
+        {
+            printf("Cmds\tMode\tSpeed\tAngle\tOri\n");
+            printf("\t%c\t%d\t%d\t%d\n", mode, speed, angle, wheel_orientation);
+        }
+    };
+
+    struct motor_arguments
+    {
+        float speed = 0;
+        float angle = 0;
     };
 
     struct leg_arguments
     {
-        struct motor_arguments
-        {
-            motor_arguments(float speed = 0, float angle = 0) : speed(speed), angle(angle)
-            {
-            }
-            float speed;
-            float angle;
-        };
-        motor_arguments steer{5}; //setting hub speed to a constant 5
-        motor_arguments hub{};
+        motor_arguments steer{motor_arguments{5, 0}};
+        motor_arguments hub;
```

Comment:
Cleaner way to do what you were trying to do. It's not a best practice since you should probably implement the big 5 constructors but I think this is okay since it's more concise and is a pretty basic data structure.

Reply:
e.g. SJSURoboticsTeam/urc-central-2021/issues/243

Reply:
Oh I see, I thought the big 5 only applied to the copy constructor rather then the contractor itself. You're saying the big 5 includes the basic constructor right?